README.md

```diff
@@ -295,14 +295,19 @@ The `task_state` field takes one of the following values
 (different queue types support different
 sets of `task_state` values, so this is a superset):
 
-* 'r' - the task is ready for execution (the first consumer executing
+* 'r' - the task is **ready** for execution (the first consumer executing
 a `take` request will get it)
-* 't' - the task has been taken by a consumer
-* '-' - the task has been executed (a task is removed from the queue
-after it
-   has been executed, so this value will rarely be seen)
-* '!' - the task is buried (disabled temporarily until further changes)
-* '~' - the task is delayed for some time
+* 't' - the task has been **taken** by a consumer
+* '-' - the task has been **executed (done)** (a task is removed from the queue
+after it has been executed, so this value will rarely be seen)
+* '!' - the task is **buried** (disabled temporarily until further changes)
+* '~' - the task is **delayed** for some time.
+
+For better understanding of the task life cycle, below is the task state
+diagram with indication of all transitions between the tasks states mentioned
+above and triggers of these transitions.
+
+![Task state diagram](images/task_state_diag.png)
```

Comment:
It might be better to do it in a separate section (#Statechart). I don't know how it would look better. What do you think?

Reply:
Put the diagram in a separate section `Task state diagram`. Also, added reference links to the sections of the methods that trigger state transitions.